primitives/avn-common/src/event_types.rs

```diff
@@ -0,0 +1,757 @@
+use codec::{Decode, Encode};
+use hex_literal::hex;
+#[cfg(feature = "std")]
+use serde::{Deserialize, Serialize};
+use sp_core::{H160, H256, H512, U256};
+use sp_runtime::{scale_info::TypeInfo, traits::Member, DispatchResult};
+use sp_std::{convert::TryInto, vec::Vec};
+
+// ================================= Events Types ====================================
+
+const WORD_LENGTH: usize = 32; // basic word type for Ethereum is 32 bytes
+const HALF_WORD_LENGTH: usize = 16; // needed for creating a u128
+const TWENTY_FOUR_BYTES: usize = 24; // needed for creating a u64
```

Comment:
This comment seems odd. A u64 is 8 bytes. At any rate, it's always smaller than a u128. Why then would it need 24 bytes?

Reply:
Please see usage of that constant.